packs/datadog/actions/run.py

```diff
@@ -0,0 +1,14 @@
+import importlib
+from st2actions.runners.pythonrunner import Action
+
+
+class ActionWrapper(Action):
+    def __init__(self, config):
+        self.config = config
+
+    def run(self, **kwargs):
```

Comment:
Ideally you would explicitly declare all the supported arguments instead of using kwargs.